cmdstanpy/stanfit.py

```diff
@@ -171,42 +174,34 @@ def chain_ids(self) -> List[int]:
 
     @property
     def cmds(self) -> List[str]:
-        """Per-chain call to CmdStan."""
+        """List of call(s) to CmdStan, one call per-chain."""
         return self._cmds
 
     @property
     def csv_files(self) -> List[str]:
-        """
-        List of paths to CmdStan output files.
-        """
+        """List of paths to CmdStan output files."""
         return self._csv_files
 
     @property
     def stdout_files(self) -> List[str]:
-        """
-        List of paths to CmdStan stdout transcripts.
-        """
+        """List of paths to CmdStan stdout transcripts."""
         return self._stdout_files
 
     @property
     def stderr_files(self) -> List[str]:
-        """
-        List of paths to CmdStan stderr transcripts.
-        """
+        """List of paths to CmdStan stderr transcripts."""
         return self._stderr_files
 
     def _check_retcodes(self) -> bool:
-        """True when all chains have retcode 0."""
+        """Returns ``True`` when all chains have retcode 0."""
         for i in range(self._chains):
             if self._retcodes[i] != 0:
                 return False
         return True
```

Comment:
```suggestion
        return all(self._retcodes)
```

There is no need to change this, but I think this does the same thing

Reply:
flip the logic - wouldn't that be "not all"?
because all retcodes need to be 0 - the long way is easier to read and reason about.